src/pkg/manifests/system:zones:brand:pkgsrc.p5m

```diff
@@ -14,17 +14,19 @@
 #
 
 set name=pkg.fmri value=pkg:/system/zones/brand/pkgsrc@$(PKGVERS)
+set name=pkg.summary value="Image Packaging System branded zone - pkgsrc zones"
 set name=pkg.description \
     value="A branded zone whose software is managed using the pkgsrc, and where some filesystems are shared with the global zone."
-set name=pkg.summary \
-    value="Image Packaging System branded zone - pkgsrc zones"
 set name=variant.arch value=$(ARCH)
-dir path=usr/lib
-dir path=usr/lib/brand
-dir path=usr/lib/brand/pkgsrc
+dir  path=etc
+dir  path=etc/zones
+file path=etc/zones/OMNIpkgsrc.xml
+dir  path=usr/lib
+dir  path=usr/lib/brand
+dir  path=usr/lib/brand/pkgsrc
 file path=usr/lib/brand/pkgsrc/config.xml mode=0644
 file path=usr/lib/brand/pkgsrc/platform.xml mode=0644
 file path=usr/lib/brand/pkgsrc/postinstall mode=0755
-license cr_Oracle license=cr_Oracle
 link path=usr/lib/brand/pkgsrc/profile.xml target=../sparse/profile.xml
-depend fmri=system/zones/brand/sparse@$(PKGVERS) type=require
+license cr_Oracle license=cr_Oracle
```

Comment:
still oracle ?

Reply:
I'm not changing that, it's just moved from a different line. All of these brands inherit from code that has that Oracle licence.